Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Coupled-Transmons/Use Case 4 - Single- and Two-Qubit State and Process Tomography/README.md

```diff
@@ -0,0 +1,185 @@
+* (optional) If the qubit pulses are relatively short (< 100ns), then the use of optimal DRAG waveforms can enhance the fidelity of the gates.
+* A two-state readout discriminator, one for each qubit, needs to be calibrated by performing readout fidelity "IQ blob" measurements, and the discriminator thresholds as well as the discriminator rotation angles, set.
+* Find the DC flux offset value of the tunable coupler such that the coupling between the two qubits in question, is off. We want the coupling between the qubits to be off, by default.
+
+Note that the state preparation and/or measurement "parts" of the state/process tomography, in the current tomography schemes, do not make use of the flux tunability of the qubits nor that of the tunable coupler, beyond parking the qubits at constant DC offsets. This is due to the fact that only separable two-qubit operations (i.e. two separate $\pi$ or $\pi/2$ pulses) are applied to prepare and/or measure the qubits' state. However, the user may wish to analyse an entangled state/pulse 'gate' operation'. In such cases, the required qua commands, which may use different constant DC fluxes or flux pulses as part of the subroutine to create the entanglement, need to be entered into the "prepare_state"/"analysed_process" functions of the scripts.
+
+
+## 2. [The configuration](configuration.py)
+
+
+### 2.1 The elements
+The configuration consists of 4 elements:
+* `rr1` and `rr2` send readout pulses and measure the transmitted signal for each qubit
+* `q1_xy` and `q2_xy` send drive XY pulses to each qubit
+* `q1_z` and `q2_z` send control Z pulses and DC voltage to the flux line of each qubit
+* `tc12` sends control pulses and DC voltage to the flux loop of the tunable coupler
+
+
+### 2.2 The operations, pulse and waveforms
+
+The readout resonators `rr1` and `rr2` each have a 'readout' operation, each of which is a constant pulse with calibrated readout lengths and amplitudes.
+
+The qubits elements `q1_xy` and `q2_xy` have several operations defined that correspond to X/2, Y/2, -X/2, -Y/2, X and Y gates. The envelopes are Gaussian (although this can be changed by the user), with optional Gaussian DRAG corrections, parameters (amplitudes, lengths, Gaussian sigmas, etc.), that need to be calibrated.
+
+The qubits' flux elements, `q1_z` and `q2_z`, have a single "zero_flux" operation that is not used. Instead, the constant DC offsets, which set the qubits at their flux sweet spots throughout an entire QUA program, need to be calibrated and the values assigned to 'max_frequency_point_q1' and 'max_frequency_point_q2'.
```

Comment:
Maybe let's add "the constant DC offsets (that are configured at the controllers section)" ?